composer.json

```diff
@@ -103,7 +103,7 @@
         "doctrine/dbal": "Required to rename columns and drop SQLite columns (~2.4).",
         "fzaninotto/faker": "Required to use the eloquent factory builder (~1.4).",
         "guzzlehttp/guzzle": "Required to use the Mailgun and Mandrill mail drivers (~5.3|~6.0).",
-        "iron-io/iron_mq": "Required to use the iron queue driver (~2.0).",
+        "iron-io/iron_mq": "Required to use the iron queue driver (~2.0|~4.0).",
```

Comment:
Fixed in 44217c15731b10ba1b1b73f06a5ce09e69c96065 Sorry, didn't realize there was another composer file for queues.